utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/client/CachedConfluentSchemaRegistryClient.scala

```diff
@@ -65,6 +66,14 @@ class CachedConfluentSchemaRegistryClient(val client: CSchemaRegistryClient, cac
     val rawSchema = client.getSchemaById(id)
     SchemaWithMetadata(new SchemaMetadata(id, unknownVersion, rawSchema.schemaType(), rawSchema.references(), rawSchema.canonicalString()), config)
   }
+
+  override def getLatestSchemaId(topic: String, isKey: Boolean): Validated[SchemaRegistryError, Int] = {
```

Comment:
isn't it enough to invoke getAllVersions(...).max?

Reply:
I think it would return versionId, not schemaId, but i like the idea - I'll see what can be done

Reply:
Curently there is no cache'able method which that transforms pair (`subject`, `schemaVersion`) to `SchemaMetadata` or `SchemaID` 
It would require futher develoment.

Reply:
:+1: let's leave it... I forgot it's version and not schema id :)